torch/_inductor/cudagraph_utils.py

```diff
@@ -10,6 +10,11 @@
 perf_hint_log = torch._logging.getArtifactLogger(__name__, "perf_hints")
 
 
+InputType = List[Union[torch.Tensor, int]]
```

Comment:
nit.. I would think `InputType` would be the singular `Union[torch.Tensor, int]`. List[InputType] is more clear maybe? If `List[Union[torch.Tensor, int]]` is `InputType` what is `Union[torch.Tensor, int]` called ?